deploy-manage/deploy/elastic-cloud/create-an-elastic-cloud-hosted-deployment.md

```diff
@@ -1,20 +1,26 @@
 ---
+applies_to:
+  deployment:
+    ess: ga
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud/current/ec-create-deployment.html
+  - https://www.elastic.co/guide/en/cloud/current/ec-prepare-production.html
+  - https://www.elastic.co/guide/en/cloud/current/ec-configure-deployment-settings.html
+  - https://www.elastic.co/guide/en/cloud-heroku/current/ech-configure-deployment-settings.html
 ---
 
 # Create an Elastic Cloud Hosted deployment [ec-create-deployment]
 
 An Elastic Cloud deployment includes Elastic Stack components such as Elasticsearch, Kibana, and other features, allowing you to store, search, and analyze your data. You can spin up a proof-of-concept deployment to learn more about what Elastic can do for you.
 
 ::::{note}
-To explore Elasticsearch Service and its solutions, create your first deployment by following one of these [getting started guides](https://www.elastic.co/guide/en/starting-with-the-elasticsearch-platform-and-its-solutions/current/getting-started-guides.html). If you are instead interested in serverless Elastic Cloud, check the [serverless documentation](https://docs.elastic.co/serverless).
+To explore {{ecloud}} and its solutions, create your first deployment by following one of these [getting started guides](https://www.elastic.co/guide/en/starting-with-the-elasticsearch-platform-and-its-solutions/current/getting-started-guides.html). If you are instead interested in serverless Elastic Cloud, check the [serverless documentation](https://docs.elastic.co/serverless).
```

Comment:
these guides don't exist in a folder together anymore. maybe point to the solution getting started guides?

Reply:
I've removed this note and added links to the solution pages in a new step of the procedure that was missing (about choosing a solution view)